docs/pages/_app.js

```diff
@@ -140,7 +144,32 @@ function PersistState() {
     window.ga('set', 'dimension2', options.userLanguage);
   }, [options.userLanguage]);
 
-  return null;
+  React.useEffect(() => {
+    /**
+     * @type {null | MediaQueryList}
+     */
+    let matchMedia = null;
+
+    /**
+     * Based on https://developer.mozilla.org/en-US/docs/Web/API/Window/devicePixelRatio#Monitoring_screen_resolution_or_zoom_level_changes
+     * Adjusted to track 3 or more different ratios
+     */
+    function trackDevicePixelRation() {
+      window.ga('set', 'dimension3', window.devicePixelRatio);
+
+      matchMedia = window.matchMedia(`(resolution: ${window.devicePixelRatio}dppx)`);
```

Comment:
Would it make sense to leverage `useMediaQuery()`?

Reply:
We do not care about the state here and `useMediaQuery` has multiple issues. I'd rather not use it unless we do need it.